spec/unit/type/user_spec.rb

```diff
@@ -457,7 +457,7 @@ def check_valid_shell; end
     end
   end
 
-  describe "when purging ssh keys" do
+  describe "when purging ssh keys", :if => Puppet::Type.type(:ssh_authorized_key) do
```

Comment:
Should these tests be moved to the module?

Reply:
Yes, they should. That came up in chat, but hasn't been done yet.